resources/scripts/components/dashboard/forms/SetupTwoFactorModal.tsx

```diff
@@ -75,41 +86,50 @@ const SetupTwoFactorModal = () => {
                     .matches(/^(\d){6}$/, 'Authenticator code must be 6 digits.'),
             })}
         >
-            {recoveryTokens.length > 0 ?
+            {recoveryTokens.length > 0 ? (
```

Comment:
You don't need the `(` here, and it goes against the code style for the software. Also, why was all of this reformatted?

Reply:
Sorry, VSCode did that for me. My bad, will fix.